xml/depl_nodes.xml

```diff
@@ -150,6 +156,94 @@
    </para>
   </listitem>
  </orderedlist>
+ <sect1 xml:id="sec-depl-ostack-designate">
+  <title>Deploying Designate</title>
+  <para>
+    &desig; provides &productname; DNS as a Service (DNSaaS). It is used to
+    create and propagate zones and records over the network using pools of DNS
+    servers. Deployment defaults are in place, so not much is required to
+    configure &desig;. &o_netw; needs additional settings for integration with
+    &desig;, which are also present in the <literal>[designate]</literal> section in &o_netw; configuration.
+  </para>
+  <para>
+    The &desig; barclamp relies heavily on dns barclamp and expects
+    it to be applied without any failures.
+  </para>
+  <note>
+   <para>
+    In order to deploy &desig;, at least one node is necessary in the DNS
+    barclamp that is not the admin node. The admin node is not added to the
+    public network. So another node is needed that can be attached to the public
+    network and appear in the designate default pool.
+
```

Comment:
If you want this to be separated into two paragraphs, close the `</para>` tag here, and reopen on the next line. This break will not display otherwise.

Reply:
Done